contrib/devtools/trailing-comment.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
```

Comment:
Missing license.

Suggest to use something like the license at end of docstring of clang-format.py .